dom/schema/v1/tollgate_crossing.sql

```diff
@@ -0,0 +1,9 @@
+create table if not exists tollgate_crossing
+(
+    id          text not null primary key,
+    tollgate_id text,
+    driver_id   text,
+    crossing    jsonb,
+    created_at  timestamptz,
+    constraint fk_tollgate foreign key (tollgate_id) references tollgate (id) on delete CASCADE
```

Comment:
I would suggest:
```sql
create table if not exists worker_tallgate
(
    recnum      bigserial not null primary key,
    worker      text not null,
    gate        text not null,
    fare        text not null,
    enter       timestamptz not null,
    leave       timestamptz    
);
```

Reply:
i agree removing FK constraint as decoupling measure, we can shift that integrity checks to the code-level.

* enter/leave. we only have enter timestamp - the time when crossing detected. 
* fare - all payment related attributes will belong elsewhere (payment module/context as we agreed)   

is worker_tollgate name really preferable over tollgate_crossing?
Crossing term used in tollgate module to represent the fact of driver crossing the gate-line.  worker_tollgate name looks more like bridge table connecting two entities, and misses the crossing sense in it. :-/

Reply:
Regarding the FK. We will properly reference all relationships when the DB schema will more or less stable. For now, we can omit FK refs.